R/import.R

```diff
@@ -100,10 +100,8 @@ import_objs <- function(
   } else {
     env <- new.env()
     files <- dir(source, pattern = "\\.[r|R]$", full.names = TRUE)
-    source_text <- list() # The contents of each file
     for (file in files) {
       source(file, local = env, keep.source = TRUE)
-      source_text[[file]] <- readLines(file)
```

Comment:
With the old code, the `source`-ing of the files is right next to the `readLines()`. This is good because it keeps the running of the code and reading of the code together. With the new version, the reading of the files happens somewhere else completely. Unless there's a really good reason for that, I'd prefer to keep things as they were.

Reply:
That makes sense. I reverted in [d7cf325](https://github.com/wch/staticimports/pull/21/commits/d7cf3251995bccff3eedeb4626a9f891eaa0f6d7).